pkg/roachprod/roachprod.go

```diff
@@ -783,16 +782,23 @@ func UpdateTargets(
 	if err := LoadClusters(); err != nil {
 		return err
 	}
-	c, err := newCluster(l, clusterName, clusterSettingsOpts...)
+	return updatePrometheusTargets(ctx, l, clusterName, clusterSettingsOpts...)
+}
+
+// updatePrometheusTargets updates the prometheus instance cluster config. Any error is logged and ignored.
+func updatePrometheusTargets(
+	ctx context.Context,
+	l *logger.Logger,
+	clusterName string,
+	clusterSettingsOpts ...install.ClusterSettingOption,
+) error {
+	// use the cluster name without the node suffix. This ensures that we update the target with details of all nodes
+	cn := strings.Split(clusterName, ":") // no error check. cluster name will either have : or not, we use index 0
```

Comment:
nit: same here, capitalize the first char of the sentence and consider moving this up above in one long-form comment, instead of in-line.